### PR TITLE
Sync Test Build

### PR DESCRIPTION
**Pull request by simonrozsival**
## What's this?
This branch will be used to create a test build for iOS and Android which will include the new sync algorithm.

### Important
_THIS TEST BUILD WILL TARGET THE STAGING SERVER_

### Relationships
Ref #7439 

## Why do we want this?
We want to be able to test the sync algorithm against the staging server in a production build on TestFlight and in Play store beta channel.

## :squid: Permissions
🚫

### CODE REVIEW

File: Toggl.iOS/Info.plist [hunk 1]
```diff
@@ -90,5 +90,10 @@
 	<array>
 		<string>en</string>
 	</array>
+	<key>NSAppTransportSecurity</key>
+    <dict>
+        <key>NSAllowsArbitraryLoads</key>
+        <true/>
+    </dict>
```
Comment (pdrj) [inline Toggl.iOS/Info.plist:97]: Is this here for a reason, or a testing leftover?

Comment (simonrozsival) [reply Toggl.iOS/Info.plist:97]: 👀 that's a leftover from testing against localhost. Thanks for noticing!

**Comment by simonrozsival** [comment]
I just triggered a TestFlight build: https://app.bitrise.io/build/63bdaa56881350e0#?tab=log

**Comment by simonrozsival** [comment]
The branch is now up-to-date with `develop` and `sync-team/sync`. I'm starting a first RC build for 2.16: https://app.bitrise.io/build/74e06bb9e6fe552e#?tab=log

**Pull request merged**